app/templates/components/forms/wizard/attendee-step.hbs

```diff
@@ -92,11 +92,13 @@
       </div>
     </div>
   {{/if}}
-  <Forms::Wizard::CustomFormInput
-    @newFormField={{this.data.newFormField}}
-    @customForms={{this.data.customForms}}
-    @event={{this.data.event}}
-    @form="attendee" />
+  {{#if this.data.event.isTicketFormEnabled}}
+   <Forms::Wizard::CustomFormInput
+     @newFormField={{this.data.newFormField}}
+     @customForms={{this.data.customForms}}
+     @event={{this.data.event}}
+     @form="attendee" />
+  {{/if}}
```

Comment:
Why add one more if and nor just move it inside the if above?

Reply:
ok